core-rust/core-api-server/src/core_api/conversions/receipt.rs

```diff
@@ -711,7 +704,7 @@ pub fn to_api_costing_parameters(
         xrd_usd_price: to_api_decimal(&engine_costing_parameters.usd_price),
         xrd_storage_price: to_api_decimal(&engine_costing_parameters.state_storage_price),
         xrd_archive_storage_price: to_api_decimal(&engine_costing_parameters.archive_storage_price),
-        tip_percentage: to_api_u16_as_i32(transaction_costing_parameters.tip_percentage),
+        tip_percentage: tip_percentage_decimal.try_into().unwrap_or(i32::MAX),
```

Comment:
Should we `unwrap`?

Reply:
Have any better ideas if the tip basis points as a percentage are larger than `i32::MAX`?